### DISCUSSION TITLE
Details on LinearTruncatedKernel for Multi-Fidelity Models

### DISCUSSION THREAD

**Discussion by roussel-ryan** [meta-pytorch/botorch#947] [Q&A]
I've been looking closely at the kernel used for SingleTaskMultiFidelityGP and the associated papers that are supposed to describe the kernel used https://arxiv.org/pdf/1903.04703.pdf. However, the default kernel used for the model is the LinearTruncatedKernel, which I cannot seem to find details about in the referenced papers. Is there another reference that can describe the motivation/explanation for the terms in this new kernel, or is it an approximation of the kernels described in the reference? The docstring provides some details, but it's not clear what bias and non-bias terms are. Thanks

**Comment by Balandat** [comment]
Hmm I am actually not sure about where exactly this comes from (if anywhere), it's been a while since we implemented this... Maybe @VilockLi or @bkarrer have relevant references to point to?
